### PR TITLE
Nullable: System.Runtime.Intrinsics

### PR DESCRIPTION
**Pull request by tannergooding**
CC. @safern, @krwq, @buyaa-n, @stephentoub 

These were relatively easy since they are end of the stack and don't actually involve any nullable types (outside the Equals and ToString overrides). Still, it covers 50 files and ~1500 APIs, so it should boost the numbers a bit 😄

### CODE REVIEW

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Arm64/Aes.PlatformNotSupported.cs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
```
Comment (safern) [inline src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Arm64/Aes.PlatformNotSupported.cs]: We try and not add an extra line after this to whenever we want to flip the switch to the whole assembly make the grep/replace easier, but I guess it is doable this way as well.

Comment (tannergooding) [reply src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Arm64/Aes.PlatformNotSupported.cs]: Fixed. I had just gone with what felt natural/readable 😄

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs [hunk 2]
```diff
@@ -189,7 +190,7 @@ public string ToString(string format)
         /// <param name="formatProvider">The format provider used to format the individual elements of the current instance.</param>
         /// <returns>An equivalent string representation of the current instance.</returns>
         /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
-        public string ToString(string format, IFormatProvider formatProvider)
+        public string ToString(string format, IFormatProvider? formatProvider)
```
Comment (stephentoub) [inline src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs]: Why can't `format` be null?

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector256_1.cs [hunk 3]
```diff
@@ -190,7 +191,7 @@ public string ToString(string format)
         /// <param name="formatProvider">The format provider used to format the individual elements of the current instance.</param>
         /// <returns>An equivalent string representation of the current instance.</returns>
         /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
-        public string ToString(string format, IFormatProvider formatProvider)
+        public string ToString(string format, IFormatProvider? formatProvider)
```
Comment (stephentoub) [inline src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector256_1.cs]: Ditto.

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector64_1.cs [hunk 4]
```diff
@@ -144,7 +145,7 @@ public string ToString(string format)
         /// <param name="formatProvider">The format provider used to format the individual elements of the current instance.</param>
         /// <returns>An equivalent string representation of the current instance.</returns>
         /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
-        public string ToString(string format, IFormatProvider formatProvider)
+        public string ToString(string format, IFormatProvider? formatProvider)
```
Comment (stephentoub) [inline src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector64_1.cs]: Ditto... I'll stop asking about these.

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs (hunk 2 above)
Comment (tannergooding) [reply src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs]: Good catch. Fixed.

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector256_1.cs (hunk 3 above)
Comment (tannergooding) [reply src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector256_1.cs]: Fixed.

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector64_1.cs (hunk 4 above)
Comment (tannergooding) [reply src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector64_1.cs]: Fixed.

**Pull request merged**